### ISSUE TITLE
Missing FSPIOP-Source - results in error code 2001 instead 3102

### ISSUE THREAD

**Issue by IvtVasquez** [mojaloop/project#1245]
**Summary**:
Error code 2001 is given instead of error code 3102 when running GET /parties request with missing FSPIOP-Source.
The following information is presented with:
- error code 2001 instead of 3102 
-"errorDescription": "Internal server error - child \"fspiop-source\" fails because [fspiop-source is required]" instead of 'Missing mandatory element - fspiop-source'

**Severity**:
High

**Priority**:
Medium

**Expected Behavior**
The expected error code when is Missing the FSPIOP Source is “error code: 3102” and description  'Missing mandatory element - fspiop-source'

**Steps to Reproduce**
1. Access the following request GET -partiesBy Type and ID --> Negative TEst cases MissingFSPIOP Source
2. Send the Get request > MissingFSPIOP Source , with missing fspiop-source
3. Error code expected “error code: 3102” and description  'Missing mandatory element - fspiop-source'

**Specifications**

  - Component (if known): ALS
  - Version: Helm v8.8.0
  - Platform:
  - Subsystem:
  - Type of testing:
  - Bug found/raised by: 

**Notes**:

- Severity when opened: High 
- Priority when opened: Medium

**Comment by elnyry** [comment]
Validated on AWS 'dev2`. Sent a GET /parties/{TYPE}/{ID} with missing FSPIOP-Source header..

```
{
    "errorInformation": {
        "errorCode": "3102",
        "errorDescription": "Missing mandatory element - fspiop-source is required"
    }
}
```

**Comment by elnyry** [comment]
This is available with ALS v9.2.3: https://github.com/mojaloop/account-lookup-service/releases/tag/v9.2.3
